lib/traixroute/detector/detection_rules.py

```diff
@@ -361,26 +401,28 @@ def check_edges(self, rule, path_asn, current, str_to_chk, ixp_long, ixp_short):
             c) current: The current hop in the path.
             d) str_to_chk: The condition keyword of the rule.
             e) ixp_long,ixp_short: The long and short IXP names.
-        Output: 
+        Output:
             True if the condition is satisfied, False otherwise.
         '''
-        
+
         [final1, final2] = self.find_numbers(rule, str_to_chk, current, False)
 
         if final1 == '' or final2 == '':
             return True
-        if self.is_int(final1) and self.is_int(final2) and 'AS_M' in str_chk:
-            if (final1 == final2 and path_asn[current - 1] != path_asn[current + 1]) or (final1 != final2 and path_asn[current - 1] == path_asn[current + 1]):
+        if self.is_int(final1) and self.is_int(final2) and 'AS_M' in str_to_chk:
+            if (final1 == final2 and path_asn[current - 1] != path_asn[current + 1]) or (
+                    final1 != final2 and path_asn[current - 1] == path_asn[current + 1]):
                 return False
-        elif self.is_int(final1) and self.is_int(final2) and 'IXP_IP' in str_chk:
-            string_hanlde = string_handler.string_handler()
```

Comment:
`string_hanlde` typo which was used as `string_handle` below

Reply:
I wonder why with this typo code still works